solver/src/solver/http_solver.rs

```diff
@@ -219,7 +219,11 @@ impl HttpSolver {
             orders: order_models,
             amms: amm_models,
             metadata: Some(MetadataModel {
-                environment: Some(self.network_id.clone()),
+                environment: Some(format!(
+                    "{} / {}",
+                    self.network_id.clone(),
+                    self.chain_id.clone()
```

Comment:
Note that for Mainnet, Rinkeby and xDai, `network_id == chain_id` AFAIK.

Reply:
I'm getting "Ethereum" as network_id currently. Are you saying I will get "Ethereum / Ethereum" for all of those networks with this change? If so I must really revise this :)

Reply:
I recall I coded [this](https://github.com/gnosis/gp-v2-services/blob/ae3d2387cf0bc464cab38fdffb62b3f3263d7165/shared/src/network.rs) a while ago. Maybe I could make it more informative and just use network_name here ?

Reply:
I changed the code to use that function instead.

Reply:
To me, the network ID is the result of `net_version` RPC call and the chain ID is the result of the `eth_chainId` RPC call. They can be different, but for all networks we support they are the same (1, 4, and 100 respectively).